Updates spacing field to double in proto

magnesj opened the pull request:
Closes #12758

magnesj committed:
Updates spacing field to double in proto

Changes the `spacing` field type from int32 to double in the`CreateMultipleFracRequest` protobuf definition

Also corrects the protobuf message name being used in the python code.

magnesj committed:
Add pytest test_create_multiple_fractures

magnesj committed:
Python code linting changes detected by black

magnesj committed:
Merge remote-tracking branch 'origin/python-black-patches-ti254z3' into 12758

kriben commented on GrpcInterface/Python/rips/tests/test_commands.py:
```diff
@@ -27,3 +27,23 @@ def test_create_lgr_well(rips_instance, initialize_test):
+        base_layer=30,
+        spacing=100.0,
+        action="REPLACE_FRACTURES",
+    )
```
Assert that the fractures were actually created?

magnesj replied:
Fixed

kriben reviewed (approved):
LGTM!

magnesj committed:
Adds fracture existence assertions

Adds assertions to verify the existence of fractures before and after creating them.
This ensures that the fracture creation functionality is working as expected.

magnesj committed:
Imports missing rips module

The test suite was failing due to a missing import statement
for the `rips` module. This commit adds the import statement,
resolving the issue.
